server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/utils/ErrorResponder.java

```diff
@@ -77,6 +77,11 @@ public ErrorResponder type(ErrorType type) {
         return this;
     }
 
+    public ErrorResponder message(String message, Object... args) {
+        this.message = String.format(message, args);
+        return this;
+    }
+
     public ErrorResponder message(String message) {
```

Comment:
I curious to know how this method can be called

Reply:
It is called when you don't specify var-args.

I discovered java to be clever enough to be relying on base method when there is a clash with a vararg version of it.

For the record I just re-used a log4j trick ;-)

And I kept it to avoid needless calls to String.format...

Reply:
interesting, thank you

Reply:
scala is not as clever as java on this point, so it can lead to interoperability issue. 
Not in this case as it is in the webadmin, but for core api it could be a problem.